docs/content/how-to/kubevirt/configuring-network.md

```diff
@@ -32,3 +32,29 @@ hcp create cluster kubevirt \
 In this example, the KubeVirt VMs will have interfaces attached to the networks
 for the NetworkAttachmentDefinitions network1 and network2 which reside in
 namespace my-namespace.
+
+## Using managed networks as default
```

Comment:
I'm not sure the term `managed networks` will be interpreted accurately enough.

I think what we want to communicate is someone can disable usage of the pod network and use a secondary network (multus) as the default network for the worker node.

So maybe `Using Secondary Network as Default`?